meta/lunch_and_learn.md

```diff
@@ -0,0 +1,52 @@
+### Lunch & Learn
+
+When Artsy was under 50 people, we would hold "Brown Bag" meetings. These were talks on a variety of art+science topics. As we out-grew the format, we started doing enginering "Brown Bag" meetings. Occasionally.
+
+This worked for a while, but was sparodic and mostly the same speakers. Once we moved to [a new standup format](http://artsy.github.io/blog/2015/03/23/artsy-technology-stack-2015/) it felt like time to re-vamp our brown bags too.
+
+Our format is pretty simple, every Thursday we have an hour slot at 12 (NYC time) on the calendar for someone to give a talk or presentation. I think [Alan](http://twitter.com/alanjay1/) described it well on [Professional Development at Artsy](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/):
+
+> Every Thursday, we do a Lunch-and-Learn session. Historically, we mostly showed off tech we use internally. But over about a year, most of our tech stack has been presented this way, so we also bring in engineers we know at other companies to share what they’re working on.
+
+
+---
+
+You're probably seeing this as an external developer to Artsy, who has been invited to talk.
+
+## Format
+
+The format is definitely not set in stone, however these are what has worked for us internally and for past contributors.
+
+#### Presentation
+
+Traditional conference/meetup style: You bring slides, we watch. You talk, we listen. You finish, we ask questions.
+
+We don't expect conference/meetup-level presentation, and practice. This is a great oppertunity to present new talks to new eyes. 
```

Comment:
> We don't expect conference/meetup-level presentation, and practice.

Could use some clarification – a lack of polish is okay. Love the note about the opportunity for new talks, maybe say "try new talk ideas"? 

"oppertunity"